test/query_hash_coercer_test.rb

```diff
@@ -0,0 +1,57 @@
+require_relative "test_helper"
+
+describe Committee::QueryHashCoercer do
```

Comment:
I'm doing that in some places even though the description doesn't reflect it. Also noticed now that I forgot to add tests for some. Will adapt.